pkg/edit/store_api.go

```diff
@@ -10,37 +10,88 @@ import (
 	"src.elv.sh/pkg/eval"
 	"src.elv.sh/pkg/eval/vals"
 	"src.elv.sh/pkg/parse/parseutil"
+	"src.elv.sh/pkg/store"
 )
 
 var errStoreOffline = errors.New("store offline")
 
 //elvdoc:fn command-history
 //
-// Outputs the entire command history as a stream of maps. Each map has a `id`
-// key that identifies the sequence number of the entry, and a `cmd` key that
-// identifies the content.
+// ```elvish
+// edit:command-history &dedup &newest-first
+// ```
 //
-// Use indexing to extract individual entries. For example, to extract the
-// content of the last command, do this:
+// Outputs the command history as a stream of maps unless `&cmd-only` is true. Each map has a `id`
```

Comment:
> ... unless `&cmd-only` is true

In general, avoid "unless", unless the condition after "unless" is truly an edge case.